node/test/lib/hyperbahn-cluster.js

```diff
@@ -135,21 +136,24 @@ function HyperbahnApp(opts) {
 
     var self = this;
 
-    self.relayChannel = opts.relayChannel;
-    self.egressNodes = opts.egressNodes;
-    self.hostPort = self.relayChannel.hostPort;
+    self.__relayChannel = opts.relayChannel;
+    self.__egressNodes = opts.egressNodes;
```

Comment:
What’s the motivation for this style change? Perhaps dunderbar is too aggressive; would one suffice?